packages/core-mobile/app/new/features/onboarding/components/CreatePin.tsx

```diff
@@ -25,6 +26,7 @@ export const CreatePin = ({
   onEnteredValidPin: (validPin: string) => void
 }): React.JSX.Element => {
   const ref = useRef<PinInputActions>(null)
+  const bioType = useBiometricType()
```

Comment:
does this bio type ever change? if not should we cache it somehow instead of rerunning the logic every time this hook is used?